src/pydata_sphinx_theme/assets/styles/components/_icon-links.scss

```diff
@@ -12,24 +35,6 @@
   margin-bottom: 0;
   list-style: none;
 
-  // Remove the padding so that we can define it with flexbox gap above
-  li.nav-item a.nav-link {
-    padding-left: 0;
-    padding-right: 0;
-
-    @include icon-navbar-hover;
-
-    &:focus {
-      color: inherit;
```

Comment:
This rule isn't needed anymore after #1864. Bootstrap was applying a color on focus to .nav-items descendants of .navbar-nav. 

All of the other rules in this section are taken care of the new .pst-navbar-icon class.